translations/base-de.yaml

```diff
@@ -15,13 +15,12 @@ steamPage:
 
         Der Kauf des Spiels auf Steam gibt dir Zugriff auf die Vollversion, aber du kannst die Demo auf shapez.io auch zuerst ausprobieren und dich später entscheiden!
     what_others_say: Was andere über shapez.io sagen
-    nothernlion_comment: This game is great - I'm having a wonderful time playing,
-        and time has flown by.
-    notch_comment: Oh crap. I really should sleep, but I think I just figured out
-        how to make a computer in shapez.io
-    steam_review_comment: This game has stolen my life and I don't want it back.
-        Very chill factory game that won't let me stop making my lines more
-        efficient.
+    nothernlion_comment: Dieses Spiel ist großartig - ich habe eine Menge Spaß beim Spielen,
```

Comment:
Kein Komma

Reply:
bist du sicher? https://www.contify.de/glossar/richtig-schreiben/komma-vor-und/ siehe Beispiel 2

Reply:
Haben und vergehen sind zwei Verben und die beiden zugehörigen Hauptsätze werden durch ein 'und' verbunden werden. Kein Komma

Reply:
In Beispiel 2 wird der Hauptsatz durch einen Nebensatz unterbrochen. Deswegen 2 Kommas davor und danach